### PR TITLE
added vuetify, logic for config loading

### PR DESCRIPTION
**Pull request by scramb**
# Changes
- Vuetify added
- Logic for loading Config into vuex state
- Theming of Logo in Login-Screen

## To-Do's
- remove UiKit and use vuetify everywhere

## Screenshots
2018-11-12-153126_1711x870_scrot

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 	<link rel="stylesheet" href="core/css/uikit.owncloud.css" />
 	<link rel="stylesheet" href="core/css/uikit.core.css" />
+	<link rel="stylesheet" href="../node_modules/vuetify/dist/vuetify.min.css" />
```
Comment (DeepDiver1975) [inline index.html]: should be in code bundle - right?

File: src/default.js [hunk 2]
```diff
@@ -93,8 +95,12 @@ Vue.component('drop', Drop);
                 router,
                 render: h => h(Phoenix)
             });
-            // inject custom config into vuex base state
+            // inject custom config into vuex
             store.dispatch('loadConfig', config.data)
+            // inject custom theme config into vuex
+            let theme = Axios.get(`src/themes/${config.data.theme}.json`).then(res => {
```
Comment (DeepDiver1975) [inline src/default.js]: src is not deployed - right - needs to be at a different location afaik

File: src/pages/login.vue [hunk 3]
```diff
@@ -1,23 +1,20 @@
...
-                </div>
-            </div>
-        </div>
-    </div>
+    <v-layout class="background primary">
+        <v-img
+          :src='`${this.$store.state.config.theme.logo.big}`'
+          width="50%"
+          aspect-ratio="1.96">
+          <v-flex class="theme--light grey lighten-2 pa-3 ma-3" xs4>
```
Comment (DeepDiver1975) [inline src/pages/login.vue]: these classes are not that easy to understand compared to uikit ...

File: src/pages/login.vue [hunk 4]
```diff
@@ -1,23 +1,20 @@
...
+            <h2 class="pa-2" v-translate>
+                Welcome to ownCloud
```
Comment (DeepDiver1975) [inline src/pages/login.vue]: theme the name

File: src/pages/login.vue [hunk 5]
```diff
@@ -1,23 +1,20 @@
...
+            </h2>
+            <v-flex class="pa-2" v-translate>
+                Please click the button below to authenticate with ownCloud and get access to your data.
```
Comment (DeepDiver1975) [inline src/pages/login.vue]: theme the name

**Comment by DeepDiver1975** [comment]
mind putting screenshots into the description? THX

File: index.html (hunk 1 above)
Comment (scramb) [reply index.html]: `<link rel="stylesheet" href="core/css/core.css" />`
bundled everything now

File: src/default.js (hunk 2 above)
Comment (scramb) [reply src/default.js]: is now themes/owncloud.json for example

File: src/pages/login.vue (hunk 3 above)
Comment (scramb) [reply src/pages/login.vue]: that's true but im sure this isn't the cleanest solution to make this working im just bad at this.

**Pull request closed without merging**